blog/_posts/2015-12-2-includeos-is-now-open-source.md

```diff
@@ -0,0 +1,39 @@
+# IncludeOS is now free and open source
+We've finally opened the lid on [IncludeOS](http://www.includeos.org), just in time for the [IEEE CloudCom](http://2015.cloudcom.org/) paper presentation yesterday. A preprint of the paper is [available from our repo](https://github.com/hioa-cs/IncludeOS/blob/master/doc/papers/IncludeOS_IEEE_CloudCom2015_PREPRINT.pdf). However, we've done quite a lot of work since the paper was written, so here's an update on what IncludeOS is now, and what you can expect in the near future. 
```

Comment:
One suggestion: since it's the first post, a brief introduction to @alfred-bratterud and the group might be in order so that the "we" here makes sense.  There are so many different groups working on unikernels now that a personal introduction is always nice to have.